### PR TITLE
Add LuaJIT.org.xml

### PR DESCRIPTION
**Pull request by pipboy96**

### CODE REVIEW

File: src/chrome/content/rules/LuaJIT.org.xml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<!--
+	luajit.org has a wildcard DNS record, all other subdomains are mismatched.
+
+	Incomplete certificate chain:
+		- wiki
```
Comment (J0WI) [inline src/chrome/content/rules/LuaJIT.org.xml]: I got a mismatch

**Comment by J0WI** [comment]
Ref #15750

File: src/chrome/content/rules/LuaJIT.org.xml (hunk 1 above)
Comment (pipboy96) [reply src/chrome/content/rules/LuaJIT.org.xml]: https://whatsmychaincert.com/?wiki.luajit.org

Comment (J0WI) [reply src/chrome/content/rules/LuaJIT.org.xml]: Both is true. It's an incomplete, mismatching cert.

Comment (pipboy96) [reply src/chrome/content/rules/LuaJIT.org.xml]: ✅

**Pull request merged**